File: 1-easy/problems/1748-sum-of-unique-elements/solution.ts
function sumOfUnique(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: int
  """
  return sum(x for x, c in collections.Counter(nums).iteritems() if c == 1);
}

export { sumOfUnique };

File: 1-easy/problems/1748-sum-of-unique-elements/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def sumOfUnique(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        return sum(x for x, c in collections.Counter(nums).iteritems() if c == 1)
